IFC4x3/Templates/Object Composition/Aggregation/Linear Composition/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Linear Composition" UniqueId="562f6342-e8e3-4551-ac5d-0d2724639975" Type="IfcLinearPositioningElement">
+	<Rules>
+		<DocModelRuleAttribute Name="Decomposes">
+			<Rules>
+				<DocModelRuleEntity Name="IfcRelAggregates">
+					<Rules>
+						<DocModelRuleAttribute Name="RelatingObject" Identification="RelatingObject">
+							<Rules>
+								<DocModelRuleEntity Name="IfcProject">
```

Comment:
Comparing with Spatial Composition and reading the documentation ("The linear structure is a hierarchical tree of linear positioning elements ultimately assigned to the project"), Should  there also be an IfcRelAggregates.RelatingObject => IfcLinearPositioningElement.IsDecomposedBy?

Reply:
Definitely wanted to talk about this. Not sure. I think decomposing alignments is one type of concept that could be explored.

Positioning alignments on stations or alignments is another one that is not on this level but still worth a discussion.

Reply:
> Should there also be an IfcRelAggregates.RelatingObject => IfcLinearPositioningElement.IsDecomposedBy?

I believe we should start a new issue / pull request to discuss this.